git-config/hooks/pre-commit

```diff
@@ -18,4 +18,13 @@ done
 pyflakes3 /usr/local/ub_tools/cronjobs
 
 
+# Format C++ source files:
+readonly changed_files=$(git diff --cached --name-only --diff-filter=ACM)
```

Comment:
Can we limit this to .cc and .h files somehow? e.g. we do not want this to be applied to .py files (and others).

Reply:
Done!

Reply:
One more thing: Can we also exclude external libraries like libstemmer?

Reply:
It's not external any more as we adopted it.

Reply:
If we change the code style that also means it will be harder to diff against the original files in the future in case we want to apply updates without losing our changes

Reply:
But i guess since we will never do that it should be ok anyway